### ISSUE TITLE
NTR: Light exposure at inappropriate hours (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7839]
New term request Light exposure at inappropriate hours (segal_140330150940):

**Comment by pnrobinson** [comment]
@MickeySegal

Would it be better to entitle this term "Excessive nocturnal light exposure"
Otherwise it is not clear what inappropriate means

see https://pubmed.ncbi.nlm.nih.gov/34903129/

**Comment by MickeySegal** [comment]
That is good.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000308
- label: Excessive nocturnal light exposure
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:34903129
- def: Exposure to continuous light at night, including night-shift work or a nocturnal lifestyle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7839
- synonyms:
